publication/2019-10-21_announcing_ngxs_3_6/article.md

```diff
@@ -4,9 +4,11 @@
 
 ## Overview
 
+- 🦄 Ivy Support (TODO decide icon: 🦄 🌼 🍒 🌸 🎄 👀 🍀 🌿 🌈 🚀 ☘️ 🌟 ⭐️ 🎋 🏆)
```

Comment:
🍀- use it! because official

![image](https://user-images.githubusercontent.com/12021443/70269080-b9614a80-17b2-11ea-9cdc-f57851a6a60f.png)

Reply:
I do really like the Unicorn!
Ivy has been a mythical creature for such a long time!

Reply:
When disputes about emojis begin, it seems to me that it’s better not to put emoji at all.... :rofl: :rofl:

Reply:
Pick a response you like